composer.json

```diff
@@ -22,6 +22,7 @@
         "symfony/form": "^2.3.5 || ^3.0",
         "symfony/http-foundation": "^2.3 || ^3.0",
         "symfony/property-access": "^2.3 || ^3.0",
+        "symfony/security": "^2.3 || ^3.0",
```

Comment:
To sum up, if something is in the `Test` directory (as opposed to the `Tests` directory), its dependencies should be in `require`, because it is exposed to client projects.